src/main/java/com/sender/email/Email.java

```diff
@@ -0,0 +1,161 @@
+package com.sender.email;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+import javax.mail.internet.AddressException;
+import javax.mail.internet.InternetAddress;
+import java.io.*;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Scanner;
+
+@Component
+public class Email implements EmailDataAccess, Serializable {
```

Comment:
This class does a lot of things, too much I would say. Consider breaking it up by functionality: create component for storage, working with console. This way you can change each component separately, which gives you more flexibility and makes partial changes easier